src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -25,5 +20,9 @@ public List<AdvertDTO> getAdverts(@RequestParam LocalDate from, LocalDate to) {
 
         return advertService.getAdvertsByDateRange(from, to);
     }
+    @PutMapping(value = "/api/advert")
```

Comment:
Your mapping is: "/api/advert/api/advert"
But should be: "/api/advert/"

Reply:
> But should be: "/api/advert/"

or "/api/advert/{id}"